### DISCUSSION TITLE
Requesting suggestions to start contributing to spline

### DISCUSSION THREAD

**Discussion by TharinduDG** [AbsaOSS/spline-spark-agent#485] [General]
@wajda I would appreciate it if you could point me to some tasks/bugs/discussions that I can help me to understand internals of spline + spark.

Thanks.

**Comment by cerveada** [comment]
Hello, 
we selected two issue you can try to implement:

#482 option to add parameters to http header
#487 java.lang.NoSuchFieldException for CreateTableAsSelect statements

The first one is easier. It is modification of http dispatcher and adding some configuration options. The second one is about extracting the lineage data from Databricks classes. It seems like the Databricks class differ from the Spark one. In that case there have to be a special logic for Databricks as well as for normal Spark.

Choose what issue you feel like and let us know. We will try to help you should you have any problems.

**Reply by TharinduDG** [reply]
@cerveada Thanks. I'll check.
